### PR TITLE
V2.1.3 fix for IE11's not supporting to Array.includes

### PR DESCRIPTION
**Pull request by Marssssssss**
Re: cocos-creator/2d-tasks#1636

Changes:
 * fix for IE11's not supporting to Array.includes

### CODE REVIEW

File: polyfill/array.js [hunk 1]
```diff
@@ -16,4 +16,20 @@ if (!Array.prototype.find) {
 
         return undefined;
     };
-}
\ No newline at end of file
+}
+
+//
```
Comment (PPpro) [inline polyfill/array.js]: 减少没必要的注释

File: polyfill/array.js [hunk 2]
```diff
@@ -16,4 +16,20 @@ if (!Array.prototype.find) {
 
         return undefined;
     };
-}
\ No newline at end of file
+}
+
+// 
+if (!Array.prototype.includes) {
+    Array.prototype.includes = function () {
```
Comment (PPpro) [inline polyfill/array.js]: 标准的 includes 实现可以参考 MDN 文档 
https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

**Comment by PPpro** [comment]
issue 关联错了，可以改下

**Comment by Marssssssss** [comment]
欸，这个 PR 我怎么关掉了

**Pull request closed without merging**